packages/database/src/migrations/20210602041852-Rename-lesmis-permission-groups-modifies-data.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const updateLESMISReportPermissionGroup = function (db, oldName, newName) {
```

Comment:
`updatePermissionGroupName` is probably more appropriate to what's happening, but in a migration it doesn't matter too much

Personally I also like to mark functions as `async` if they return a promise, just as a flag to anyone using them. Again, in a migration, doesn't matter at all, but just for future reference